kratos/solving_strategies/convergence_accelerators/convergence_accelerator.h

```diff
@@ -53,6 +53,12 @@ class ConvergenceAccelerator
     typedef typename TSparseSpace::VectorPointerType               VectorPointerType;
     typedef typename TSparseSpace::MatrixPointerType               MatrixPointerType;
 
+    typedef typename TDenseSpace::VectorType                         DenseVectorType;
+    typedef typename TDenseSpace::MatrixType                         DenseMatrixType;
+
+    typedef typename TDenseSpace::MatrixPointerType           DenseMatrixPointerType;
+    typedef typename TDenseSpace::VectorPointerType           DenseVectorPointerType;
+
```

Comment:
hmmm
I would rather not add those
after all we don't want to encourage the usage of dense matrices
```suggestion
```

Reply:
The point is that convergence accelerators always require dense matrices. Indeed we need them to be dense as we will use these later on for SVD and QR utilities.

Reply:
QR decomposition doesnt exist for sparse matrices?

Reply:
Nope, our QR algorithm is understood to be applied only to dense matrices. Indeed the API asks for the top left corner entry, which makes no sense in the sparse case.

Reply:
we have been discussing this at lenght, and the point is that those matrices are simply dense. So i would accept this

Reply:
@rubenzorrilla which API are you referring to ? 
As I remember for eigen solvers it will not as for top left corners !

Reply:
`qr_util.h` in the core.